src/scss/cs/homepage.scss

```diff
@@ -195,7 +200,7 @@
       color: #fff;
       text-align: center;
       border-radius: 5px;
-      padding: 5px 0;
+      padding: 15px;
```

Comment:
Could we use Bootstrap for the `padding` styling instead of custom CSS?

Reply:
Because these are minor adjustments and there is a lot of things that can be done better by using bootstrap, I just feel like it is not part of the ticket to alter how it is working. 
It will not be consistent to rework just the padding and leave everything else the same. It will make the code messier.